_sass/minima/_nav-list.scss

```diff
@@ -49,8 +53,8 @@
       display: flex;
       justify-content: center;
       align-items: center;
-      width: 34px;
-      height: 34px;
+      width: 30px;
+      height: 30px;
       background-color: transparent;
       border-width: 0;
       appearance: none;
```

Comment:
```suggestion
      appearance: none;
      cursor: pointer;
```

@GBKS This will fix the issue you described where hovering the arrow does not show the pointer.

Reply:
Thank you. It's the weirdest thing, today the hand/pointer cursor shows up correctly even through I haven't changed anything... across browsers, too.

If I manage to reduce the issue again, I'll take another look, but will merge this issue for now.